frappe/public/js/frappe/router.js

```diff
@@ -139,6 +139,12 @@ frappe.router = {
 		if (!frappe.app) return;
 
 		let sub_path = this.get_sub_path();
+		if (frappe.boot.setup_complete) {
+			!frappe.re_route["setup-wizard/0"] && (frappe.re_route["setup-wizard/0"] = "app");
```

Comment:
let's not hardcode slide number here? 

Simple logic:

` if setup_complete is not true and route does not contain `setup-wizard` then redirect`